apiserver/facades/client/modelgeneration/interface.go

```diff
@@ -26,6 +26,8 @@ type Model interface {
 	AddBranch(string, string) error
 	Branch(string) (Generation, error)
 	Branches() ([]Generation, error)
+	CommittedBranch(int) (Generation, error)
```

Comment:
I would be in favour of calling these `Generation` and `Generations` in the interest of terseness, but that isn't crucial right now.